UPGRADING.md

```diff
@@ -1,5 +1,17 @@
 # Upgrading existing installations
 
+## From `2020.06.0` to `2020.10.0`
+
+Due to the significant jump in Synapse version we recommend to wipe the Synapse 
+database when upgrading.
+
+This can be done by stopping the entire RSB:
+
+    docker-compose stop
+
+And then removing the postgres data directory (`<DATA_DIR>/db`).
```

Comment:
Should we mention how to access the metrics here_

Reply:
So far the monitoring isn't documented at all. 
We should do that but it should probably be in its own document. 
Issue to track: #193